lib/circuit.js

```diff
@@ -222,6 +232,17 @@ class CircuitBreaker extends EventEmitter {
         handleError(error, this, timeout, args, resolve, reject);
       }
     });
+    if (this.options.cache) {
```

Comment:
I'm not following : ]
if the option.cache === Object then this will be true on this if statement...